libraries/AP_Tuning/AP_Tuning.cpp

```diff
@@ -202,23 +202,23 @@ void AP_Tuning::check_input(uint8_t flightmode)
 
     //hal.console->printf("chan_value %.2f last_channel_value %.2f\n", chan_value, last_channel_value);
 
+    const float dead_zone = 0.02;
     if (mid_point_wait) {
         // see if we have crossed the mid-point. We use a small deadzone to make it easier
         // to move to the "indent" portion of a slider to start tuning
-        const float dead_zone = 0.02;
         if ((chan_value > dead_zone && last_channel_value > 0) ||
             (chan_value < -dead_zone && last_channel_value < 0)) {
             // still waiting
             return;
         }
         // starting tuning
         mid_point_wait = false;
-        GCS_SEND_TEXT(MAV_SEVERITY_INFO, "Tuning: mid-point %s", get_tuning_name(current_parm));
         AP_Notify::events.tune_started = 1;
     }
     last_channel_value = chan_value;
 
     float new_value;
+    static float old_value;
```

Comment:
the static doesn't look right for when we change which parameter we are tuning, we'd need the variable in the class and reset on param change